pysat/_instrument.py

```diff
@@ -2633,8 +2658,8 @@ def _get_supported_keywords(load_func):
 
     Returns
     -------
-    list
-        list of keyword argument strings
+    dict
+        dict of keywords and values
```

Comment:
```suggestion
    out_dict : dict
        dict of keywords and values
```

But change "dict of keywords and values", which describes all dicts ever, to say what's really inside this dict.